src/scripts/main.js

```diff
@@ -1,3 +1,96 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  // максимальна кіслькість рядків і стовців
+  const maxRow = 10;
+  const maxColumn = 10;
+  const minRow = 2;
+  const MinColumn = 2;
```

Comment:
The variable `MinColumn` should be consistently named with camelCase as `minColumn` to match the naming convention used for `minRow` and avoid confusion.